keyboards/kumo/config.h

```diff
@@ -0,0 +1,30 @@
+#ifndef CONFIG_H
```

Comment:
We can replace ALL the include guards with `#pragma once`

```suggestion
#pragma once
```